services/web/server/source/async_sio.py

```diff
@@ -71,6 +74,59 @@ def connect(sid, environ):
     await SIO.emit('stopJupyter', data=result, room=sid)
 
 
+@SIO.on('presignedUrl')
+async def retrieve_url_for_file(sid, data):
+    _LOGGER.debug("client %s requests S3 url for %s", sid, data)
+    try:
+        public_url = 'play.minio.io:9000'
+        public_access_key = 'Q3AM3UQ867SPQQA43P2F'
+        public_secret_key ='zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG'
```

Comment:
I would advise strongly against storing keys
* in the code
* on a public repo ...

Reply:
Those are the very public minio url and credentials. Later on will be replaced by real data coming from the DB

Reply:
ok